package.json

```diff
@@ -43,6 +43,7 @@
     "@radix-ui/react-switch": "^1.1.0",
     "@radix-ui/react-tooltip": "^1.1.2",
     "@yext/eslint-config": "^1.0.0",
+    "@yext/pages": "^1.2.0-beta.3",
```

Comment:
Thinking about it more, I think we need document to be passed to resolveProp to avoid this lib from depending on pages.